packages/Button/stories/examples/Basic.js

```diff
@@ -78,6 +80,11 @@ const ExampleStory = () => (
       </Button>
     </p>
     <Rule />
+    <p>
+      <Button.Link onClick={clickHandler} isOpenNewTab={false} kind="primary" href="/about" text="Link" />
+      <Button.Link onClick={clickHandler} kind="secondary" href="/about" text="Link that opens in new tab" />
```

Comment:
`/about` is not a valid link. Just use google

Reply:
done